### ISSUE TITLE
Table of contents for articles

### ISSUE THREAD

**Issue by lvclark** [r-universe-org/bugs#56]
For readability, it would be nice if articles displayed with tables of contents!  Here's one of mine, where the plain HTML shows the TOC but the R-universe render does not: https://lvclark.r-universe.dev/ui#view:polyRAD/polyRADtutorial.html

**Comment by dpprdan** [comment]
There is a TOC for your article. It only shows on (very) wide screens though.

![screenshot of the article with TOC on the right side](https://user-images.githubusercontent.com/1423562/119564449-18e95680-bda9-11eb-80ea-2ef3ab3f6c9f.png)

Also the links are not working, i.e. one cannot jump to the respective headline (which is what I came here to report in the first place).

**Comment by jeroen** [comment]
> Update: Links don't work in Firefox (88) on Windows (10, 2004), but work on Chrome and Edge. Sorry, should have checked that first.

Yes I know, there is some weird behavior of iframes. If you click the raw html link at the top of the document that takes you to https://lvclark.r-universe.dev/articles/polyRAD/polyRADtutorial.html and there the links should work in any browser.
